system/kube-monitoring-admin-k3s/Chart.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
     version: "^2.5"
   - name: oomkill-exporter
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.5.11
```

Comment:
Wouldn't this break if a new version is pulled in, since no `helm dep up` is run?